docs/UserManuals/ConfigUI/webhook.md

```diff
@@ -4,21 +4,30 @@ sidebar_position: 7
 description: Config UI instruction for Webhook
 ---
 
-Visit the config-ui via the Domain Name or IP Address and Port, 
+Visit config-ui: `http://localhost:4000`.
 
-### Add a new incoming webhook
-![image](https://user-images.githubusercontent.com/3294100/191309840-460fbc9c-15a1-4b12-a510-9ed5ccd8f2b0.png)
+### Step 1 - Add Data Connections
+
+![webhook-add-data-connections](/img/ConfigUI/webhook-add-data-connections.png)
 
 #### Webhook name
+
 We recommend that you give your webhook connection a unique name so that you can identify and manage where you have used it later.
 
 ### Use Webhooks
```

Comment:
Please also update the v0.14 configuring webhook doc after this one has been updated.